src/components/core/core.js

```diff
@@ -109,7 +109,11 @@ export default class Core extends UIObject {
    * @type {Object}
    */
   get activePlaybackEl() {
-    return this.activePlayback.$el.find('video')[0] || this.activePlayback.el
+    if (!this.activePlayback) return false
+    if (this.activePlayback.el)
+      return this.activePlayback.el
+    return this.activePlayback.$el.find('video')[0]
```

Comment:
```suggestion
    if (this.activePlayback.el) return this.activePlayback.el
    return this.activePlayback.$el.find('video')[0]
```

I think this code is more readable. What do you think?